.github/workflows/audit.yaml

```diff
@@ -0,0 +1,71 @@
+name: audit
+
+on: [pull_request] 
+
+jobs:
+  audit:
+    name: Audit packages
+    runs-on: ubuntu-latest
```

Comment:
I noticed this workflow currently fails. Also same question as on Feedback-Service; do we want this workflow to be separate from the other pull-request and merge-main workflows? Or should it be one workflow?